README.md

```diff
@@ -24,7 +24,7 @@
 
 
 
-- [Live Demo Link]([https://yourdeployedapplicationlink.com](https://github.com/SSEKA/Portolio.git))
+- [Live Demo Link]([https://yourdeployedapplicationlink.com](https://github.com/SSEKA/My-Portfolio.git))
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As suggested by the previous reviewer, all the headings on the readme template must be present on your readme file except the FAQ section which is optional. At this moment, some headings are missing from your readme. Such as `Key Features and Future Features` etc. Kindly follow [Microverse template](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your professional readme.